### ISSUE TITLE
wrong URL in samples/09-download-build-use-jsoncpp/install_script.cmake

### ISSUE THREAD

**Issue by 15knots** [toeb/cmakepp#87]
set(uri "ttps://github.com/open-source-parsers/jsoncpp...
missing 'h'

**Comment by toeb** [comment]
yeah I have that fixed already.  I need to redo some samples because I changed the dependency management system quite a bit.  I will be able to merge it soon.
